index.js

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env node
 
-require('dotenv').load({silent: true});
+require('dotenv').config();
```

Comment:
We may want to keep `{silent: true}` here since `REGISTRY_SERVER` is not required for all commands and when run as a bin dependency `.env` from this project will not be present at the default path of [`path.resolve(process.cwd(), '.env')`](https://github.com/motdotla/dotenv/tree/7301ac9be0b2c766f865bbe24280bf82586d25aa#path).

Reply:
`silent` option was removed in favor of `verbose` in dotenv [v3.0.0](https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md#300---2016-12-20), so it is the new default.